app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,73 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/*примеси и переменные*/
+
+$gray-medium-light: #eaeaea;
+
+
+a{
+	color: #555;
```

Comment:
Please fix indents, I guess there problem there is tab but should be two spaces.